test/rfc7540_SUITE.erl

```diff
@@ -2955,11 +2957,10 @@ client_settings_disable_push(Config) ->
 %% (RFC7540 6.8) GOAWAY
 % @todo GOAWAY frames have a reserved bit in the payload that must be ignored.
 %
-%% @todo We should eventually implement the mechanism for gracefully
+%% The following test cases test the mechanism for gracefully
 %% shutting down of the connection. (Send the GOAWAY, finish processing
 %% the current set of streams, give up after a certain timeout.)
```

Comment:
Just remove the todo entirely.

Reply:
Where does this todo actually end? There is some quoted RFC text with another todo `@todo And it get's more complex when you think about h1 to h2 proxies.` and then some more quoted RFC text which is now implemented. Then, there is some text which is not marked as a todo, but it wasn't implemented before this PR. Should the comments about everything that is implemented be deleted?

Reply:
I'm keeping the quoted RFC text.

Reply:
The relevant documentation should be given to the `doc/1` function. It's OK if it takes 3 lines if it's something complex. Not much point keeping it in comments hidden in the middle of the test file, best to have it in log reports.

These todo comments are for things left to implement (or test), so whatever has a test shouldn't be a comment anymore. I'll double check them when merging.